pkg/controller/metadata/metadata_controller.go

```diff
@@ -54,6 +54,8 @@ const (
 	metadataCACertsKey = "metadataCACerts"
 	truststorePassword = "mashmallow"
 	VersionAnnotation  = "metadata-version"
+	ValidityDays       = "validityDays"
+	ValidUntil         = "validUntil"
```

Comment:
Can we make the naming consistent, please.
The key names should be all in camelCase: `versionAnnotation`, `validityDays` and `validUntil`